zulipterminal/ui_tools/views.py

```diff
@@ -1330,6 +1330,19 @@ def __init__(self, controller: Any, stream_id: int) -> None:
         ]
 
         total_members = len(stream["subscribers"])
+        Stream_id = stream["stream_id"]
```

Comment:
We already have the stream_id being passed as a parameter to StreamInfo view, so there's no need to re-define it again here.

Reply:
Removed.